packages/block-library/src/column/edit.native.js

```diff
@@ -111,30 +111,22 @@ function ColumnEdit( {
 		onChangeWidth( nextWidth );
 	};
 
-	const renderAppender = useCallback( () => {
-		const { width: columnWidth } = contentStyle[ clientId ];
-		const isFullWidth = columnWidth === screenWidth;
-
-		if ( isSelected ) {
-			return (
-				<View
-					style={ [
-						styles.columnAppender,
-						isFullWidth && styles.fullwidthColumnAppender,
-						isFullWidth &&
-							hasChildren &&
-							styles.fullwidthHasInnerColumnAppender,
-						! isFullWidth &&
-							hasChildren &&
-							styles.hasInnerColumnAppender,
-					] }
-				>
-					<InnerBlocks.ButtonBlockAppender />
-				</View>
-			);
-		}
-		return null;
-	}, [ contentStyle[ clientId ], screenWidth, isSelected, hasChildren ] );
+	const { width: columnWidth } = contentStyle[ clientId ];
```

Comment:
Could this be removing the performance optimization previously made here by using `useCallback`? I remember there were a few performance optimization done around here before: https://github.com/WordPress/gutenberg/pulls?q=is%3Apr+sort%3Aupdated-desc+performance+label%3A%22Mobile+App+-+i.e.+Android+or+iOS%22+is%3Amerged+label%3A%22%5BType%5D+Performance%22

Could maybe using `useRef` bring that back?

Reply:
Hey @ceyhun!
I imagine the reason for the performance issues was probably that it was creating a new function for every render when one of that dependencies had changed. It would avoid any optimization from React reconciliation. Now it should be fine.
I'd suggest keeping it as it is, and if we really see new performance issues, we could try to fix the origin of this. WDYT?

Reply:
I think now it will return a new `appender` component for every render instead. But I guess that was also happening before, even when the same `renderAppender` function was called. In that case I agree to keep it as it is.